lib/klaviyo/apis/data_privacy.rb

```diff
@@ -0,0 +1,20 @@
+module Klaviyo
+  class DataPrivacy < Client
+    DATA_PRIVACY = 'data-privacy'
+    DELETETION_REQUEST = 'deletion-request'
+
+    # Submits a data privacy-related deletion request
+    # @param identified [Hash] a hash containing either email, phone_number or person_id
+    # @return will return with HTTP OK on success
+    def self.deletion_request(kwargs = {})
+      check_required_args(kwargs)
```

Comment:
thanks for making this, we were about to start working on this!

We'd prefer something like:
kwargs = {id_type: id_type, identifier: identifier}
a check to make sure the id_type is either [email, phone_number, id]

that way we don't need to do the below and we could just pass the dictionary values over Ex. `phone_number: 5555555555`